.github/workflows/deploy-local-acs.yml

```diff
@@ -0,0 +1,168 @@
+  #           | del(. | select(.values=="7.2.N_values.yaml"))
+  #           | select(.name=="alfresco-content-services")]
+  #       run: |
+  #         echo "${JSON}" | jq -nc '${{ env.JQ_FILTER }}'
+  #         VERS=$(echo "${JSON}" | jq -nc '${{ env.JQ_FILTER }}')
+  #         echo "json=$VERS" >> $GITHUB_OUTPUT
+  helm_integration:
+    runs-on: alfrescoPub-ubuntu2204-16G-4CPU
+    timeout-minutes: 12
+    name: Deploy env (name to be updated)
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683  # v4.2.2
+
+      - uses: azure/setup-helm@fe7b79cd5ee1e45176fcad797de68ecaf3ca4814
+        with:
+          version: "3.14.3"
+
+      - name: Login to Docker Hub
+        uses: docker/login-action@9780b0c442fbb1117ed29e0efdff1e18412f7567  # v3.3.0
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+
+      - name: Login to Quay.io
+        uses: docker/login-action@9780b0c442fbb1117ed29e0efdff1e18412f7567  # v3.3.0
+        with:
+          registry: quay.io
+          username: ${{ secrets.QUAY_USERNAME }}
+          password: ${{ secrets.QUAY_PASSWORD }}
+
+      - name: Setup cluster
+        uses: Alfresco/alfresco-build-tools/.github/actions/setup-kind@v8.4.0
+        with:
+          ingress-nginx-ref: controller-v1.8.2
+          metrics: "true"
+
+      - name: Set nginx ingress config
+        run: >-
+          kubectl -n ingress-nginx patch cm ingress-nginx-controller
+          -p '{"data": {"allow-snippet-annotations":"true"}}'
+
+      - name: Create registries auth secret
+        run: >-
+          kubectl create secret generic regcred
+          --from-file=.dockerconfigjson=$HOME/.docker/config.json
+          --type=kubernetes.io/dockerconfigjson
+
+      # to be skipped imo
+      # - name: Check if cgroup v2 workaround is needed
+      #   if: startsWith(matrix.values, '7.')
+      #   env:
+      #     VALUES_FILE: ${{ matrix.values }}
+      #   id: cgroupv2-workaround-extra-values
+      #   run: |
+      #     if [ ${VALUES_FILE:2:1} -le 2 ]; then
+      #       echo "Enabling cgroupv v2 workaround"
+      #       echo "helm_install_params=--values test/cgroup-v2-workaround-values.yaml" >> "$GITHUB_OUTPUT"
+      #     fi
+
+      # - name: Check if we want additional helm customizations
+      #   id: configurable-extra-values
+      #   run: |
+      #     if [ "${{ matrix.values }}" = "pre-release_values.yaml" ] || [ "${{ matrix.values }}" = "values.yaml" ]; then
+      #       echo "Enabling clustered tests with 2 replicas"
+      #       echo "helm_install_params=--set alfresco-repository.replicaCount=2" >> "$GITHUB_OUTPUT"
+      #     fi
+
+      - name: Add dependency chart repos
+        run: |
+          helm repo add self https://alfresco.github.io/alfresco-helm-charts/
+          helm repo add elastic https://helm.elastic.co/
+
+      # Name and value to be fetched by automation??
```

Comment:
add a new step before this one using actions/checkout to clone acs-deployment as well, then adjust paths here to acs-deployment/helm/alfresco-content-services

Reply:
see https://github.com/Alfresco/alfresco-dockerfiles-bakery/blob/main/.github/workflows/reusable_build_and_test.yml#L257

Reply:
Done